Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import './App.css';
 import HeaderContainer from './components/HeaderComponents/HeaderContainer.js'
+import CardContainer from './components/CardComponents/CardContainer.js'
 
 const App = () => {
   return (
+    <div>
     <div>
```

Comment:
This would be more readable if you indented the divs containing HeaderContainer and CardContainer.